app/helpers/diffusion_history_status_helper.rb

```diff
@@ -0,0 +1,23 @@
+module DiffusionHistoryStatusHelper
```

Comment:
1. I think we should move this method to the `models/page.rb` file. Since its true goal is to get adopting facilities for the practices of a `PageMapComponent`.
2. We should consider renaming this method. Maybe something like `get_adopting_facilities`, since it will belong to the `PageMapComponent` class. We would also be able to remove the `practices_list` param, which is actually the component, not the list of practices. We would also be able to remove the `successful_adoptions`, `in_progress_adoptions`, and `unsuccessful_adoption` params, because we would be able to just use the columns directly (so something like `&& display_successful_adoptions`).
3. We should write a unit test to ensure it works properly.

Reply:
Moved the method to the page model and removed params that can be referenced by the map_component being sent in.  yeah thats a good refactor.